collectors/UcsmCollector.py

```diff
@@ -1,24 +1,23 @@
-from prometheus_client.core import GaugeMetricFamily
+from prometheus_client.core import InfoMetricFamily
 from BaseCollector import BaseCollector
 
 
 class UcsmCollector(BaseCollector):
-    def __init__(self, creds, inventory_file):
-        super().__init__(creds, inventory_file)
+    def __init__(self, creds, config):
+        super().__init__(creds, config)
 
     def describe(self):
-        yield GaugeMetricFamily("ucsm_metrics", "ucsm_collector_registered")
+        yield InfoMetricFamily("ucsm_metrics", "ucsm_collector_registered")
 
     def collect(self):
         print("UcsmCollector: Get updated handles !")
         self.get_handles()
-        g = GaugeMetricFamily('ucsm_info', 'UCSM server information',
-                              labels=['server', 'firmware_version'])
+        g = InfoMetricFamily('ucsm_info', 'UCSM server information')
```

Comment:
this should have the same name as in describe

Reply:
Changed.